### DISCUSSION TITLE
2024-06-30-china

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#186] [General]
# 置身事内 - 读书笔记 - Tw93

Hi, I'm Tw93, a Frontend Engineer from China, also dabbling a bit in Swift, Rust, Python, and Sketch. I'm the creator of MiaoYan, Pake, Trendy Weekly, XRender, and WeexUi. Believing in long-termism, I am passionate about open source and enjoy experimenting with new ideas.Study最近把「置身事内 - 中国政府与经济发展」这本书读完了，去年粗读了下，最近趁着杭州持续下雨呆家里听着雨声看看书很自在，是一本可以读两遍的好书，花了点时间把一些不错的观点摘抄备忘。

https://tw93.fun/2024-06-30/china.html

**Comment by yexia553** [comment]
去年读的这本书，用微信读书在通勤路上听了两遍，难得的好书，好在是前几年出版的，放在现在都不知道能不能出版了

**Reply by tw93** [reply]
哈哈哈 好书好好读

**Comment by Perfecto23** [comment]
写的真不错，我决定去把这本书看下

**Reply by tw93** [reply]
挺好的 这本书

**Comment by jiangyangcreate** [comment]
读完这本书了，读这本书时有种很奇怪的感觉，我无法清晰讲述当下现状，但觉得作者的解读也平平。 如同刚出版的《了不起的盖茨比》，因为贴近时代而被认为没有震撼的感觉，我想20年后，再回顾这本书，会让未来人有别样的感觉。

**Reply by tw93** [reply]
多看几本就好了

**Comment by pedyc** [comment]
很不错的书本推荐，加入书单了

**Comment by hucstar** [comment]
真大佬啊！已订阅博客。
